### DISCUSSION TITLE
SafeExamBrowser is Running, But Nowhere to be Found

### DISCUSSION THREAD

**Discussion by rhafaelc** [SafeExamBrowser/seb-win-refactoring#618] [Q&A] [answered]
This is my first time using safe exam browser for university exam. When I open SEB, it takes me back to my home screen. It only shows the login box for a couple of seconds. 

I already tried using SEB latest version and it is fixed, although it flashes every couple of seconds. But I can't use the latest version due to my university only provides the 3.0.1 version that is integrated with Zoom. I can only close the program from Task Manager. 

Is there a button that can take me back to the SEB client? or is there a way that the latest SEB version can give permission to Zoom?

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/109317539/ba682996-8bc7-4067-8d66-22b9e88eaaee)

Safe Exam Browser
Version 3.0.1(x64)
3.0.1.163

Windows 11 Home Single Language (Edition)
22H2 (Version)
22621.1702 (OS Build)

**Comment by dbuechel** [comment, marked as answer]
Your university should definitely consider updating its supported SEB version. Apart from that, there appears to be an issue with an application that is interfering with SEB's kiosk mode. We had such reports for e.g. Cisco WebEx or ESET (see #333 and #524 respectively), so try to search for any similar applications and terminate them before starting SEB.
